app/views/participants/_details.html.erb

```diff
@@ -52,10 +53,17 @@
 					<div class="small light">
 						<%= "#{enrollment.enrollment_status_name}, " unless enrollment.enrollment_status_name.blank? %>
 						<%= raw [enrollment.began_on.try(:to_s, :month_year), enrollment.ended_on.try(:to_s, :month_year)].compact.join('&ndash;') %>
+						<br/>
+						<% list_of_majors = enrollment.majors_list 
+						    majors = list_of_majors.split(", ")
+						    if majors.first.nil? || majors.first.downcase == "NOT APPLICABLE".downcase
+						    	list_of_majors = ""
+						    end %>
+						    <%=  list_of_majors %>
```

Comment:
Also this line could be moved over a few spaces to keep the indenting consistent.